### PR TITLE
Roadmap for libp2p

### PR DESCRIPTION
**Pull request by diasdavid**
This is the first pass, using the time I have to get it started, feel free to start adding/commenting stuff

### CODE REVIEW

File: ROADMAP.libp2p.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### Resources
+
+- [specs](https://github.com/ipfs/specs/tree/master/libp2p)
+
+## Milestones
+
+#### 1 - swarm interop (go+js)
+
+- [ ] Factor go-libp2p
```
Comment (jbenet) [inline ROADMAP.libp2p.md]: this depends on:
- [ ] fix multipart problem?
- [ ] fix gx?

Comment (whyrusleeping) [inline ROADMAP.libp2p.md]: multipart is fixed, now just need to update gx to the new pathing scheme.

Comment (diasdavid) [inline ROADMAP.libp2p.md]: @whyrusleeping sweet! It was a go thing, right?

**Comment by RichardLitt** [comment]
Merged on my request. We can update with small versions. @diasdavid tag me on any PRs if needed.

**Pull request merged**